src/pages/Login.tsx

```diff
@@ -47,14 +48,20 @@ const Login = () => {
       <IonContent fullscreen>
         <IonList>
           <IonItem>
-            <IonLabel>Username</IonLabel>
+            <IonLabel>
+              <p>Log into an account</p>
+            </IonLabel>
+          </IonItem>
+          <IonItem fill="outline">
+            <IonLabel position="floating">Username</IonLabel>
             <IonInput
               ref={usernameInputRef}
               placeholder="Enter username"
             />
           </IonItem>
-          <IonItem>
-            <IonLabel>Password</IonLabel>
+          <p></p>
```

Comment:
What is this empty paragraph tag for?

Reply:
This was just to add some whitespace haha. I do not know any other way to do it yet.

Reply:
Ah I see! Clever solution. Ionic provides us with some CSS utilities. In this case, instead of an empty paragraph tag, you might be able to add a class of `ion-margin-bottom` to `<IonItem fill="outline">` above so it may look like `<IonItem className="ion-margin-bottom" fill="outline">`.

Reply:
I see, thank you for letting me know, I have implemented those changes. I will continue to read up on the documentation of the ionic framework.

Reply:
Awesome. It looks like this paragraph tag might've accidentally slipped through.

Reply:
Yea, my bad I gotta check better. You were quick.